someSong/src/pages/browse-questions/browse-questions.ts

```diff
@@ -44,16 +44,15 @@ export class BrowseQuestionsPage {
   getAllEnabledQuestions(){
```

Comment:
why do we even have this function?
why not just use the `getAllUnresolvedQuestions` in the `Question` provider?
why add the `enable=true` property in the constructor, and return a new array of enabled questions?
doesn't it have the same result as `this._quetion.getAllUnresolvedQuestions`?